### PR TITLE
Add section on "Dev Docs"

### PR DESCRIPTION
**Pull request by coderkevin**
This adds information on what the devdocs are and how to view them.

No testing needed, just view the .md

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -25,3 +25,11 @@ There are also some helper scripts:
...
  - `npm run lint` : Run eslint over the javascript files
  - `npm run i18n` : Create a PHP file with the strings from the javascript files, [used to get around lack of JS support in WordPress.org](https://github.com/WordPress/packages/tree/master/packages/i18n#build).
+
+## Dev Docs
+
+There is a "devdocs" page which is useful for displaying components individually outside of the application:
+
+```http://<your-wp-site>/wp-admin/admin.php?page=wc-admin#/devdocs```
+
+This is useful for viewing of (WooCommerce)[https://woocommerce.github.io/wc-admin/#/components/] components and ad-hoc testing.
```
Comment (ryelle) [inline docs/README.md]: Looks like your link formatting is reversed, it should be `[]` around the text & `()` around the link. Could you also make the link text include the word "components"? that way it's a little more descriptive.

Comment (coderkevin) [reply docs/README.md]: Oops, will fix.

Comment (coderkevin) [reply docs/README.md]: Fixed in 093e36e. Thanks!

**Review by ryelle — APPROVED** [review]
Looks good now 👍

**Comment by ryelle** [comment]
FYI, you can also test changes to the docs locally by running `npx docsify serve docs` - it'll spin up a local server just like what's at https://woocommerce.github.io/wc-admin/

**Comment by coderkevin** [comment]
Thanks @ryelle, I added that to the document as another option.

**Comment by ryelle** [comment]
Ah, that's not what I meant– `docsify` renders the docs at https://woocommerce.github.io/wc-admin/#/components/, not the dev docs. I was just letting you know that you can test the docs markdown locally.

**Pull request merged**